src/modules/users/api/update-user-mutation.ts

```diff
@@ -2,6 +2,8 @@ import { gql, useMutation } from "@apollo/client";
 import { UpdateUserInput } from "cv-graphql";
 import { UpdateUserResult } from "./users.types";
 import { GET_USERS_QUERY } from "./get-users-query";
+import { USER } from "./get-user-query";
```

Comment:
why is it "get_users_query" but "user" (and not "get_user_query")?

Reply:
renamed that and some others